sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -28,7 +28,7 @@ git = "https://github.com/facebook/hermes.git"
 if ENV.has_key?('HERMES_ENGINE_TARBALL_PATH')
   Pod::UI.puts '[Hermes] Using pre-built Hermes binaries from local path.' if Object.const_defined?("Pod::UI")
   source[:http] = "file://#{ENV['HERMES_ENGINE_TARBALL_PATH']}"
-elsif version == '1000.0.0'
+elsif version == '1000.0.0' || version.start_with?('0.0.0-')
```

Comment:
This is going to change soon as we're looking for a place where to store hermes tarballs also for nightlies: https://github.com/react-native-community/discussions-and-proposals/pull/508

Reply:
that's super helpful where we could test nightly without building hermes from source 🎉